portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,53 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function intro() { 
+	document.getElementById("inner-content").style.display = "block"; 
+	document.getElementById("para").innerHTML = `Hi everyone! My name is
+	Michaela Murray, and I’m currently a rising junior at Stanford
+	University.  I am majoring in Computer Science with a concentration in
+	Systems, and I plan to minor in mathematics and music. So far, my
+	favorite CS class is Operating Systems, which was taught in C. I’ve also
+	enjoyed my Networking class taught in C++ and a distributed systems
+	paper-reading class where I did the final project in Rust. Last summer,
+	I did research in video analytics, and throughout this past year I have
+	continued to engage in research, the most of which was developing a
+	sensor peripheral driver for TockOS.  When I am not taking classes or
+	doing research, my main activities are being Vice President of our
+	school’s security club and taking violin lessons. A fun fact about me is
+	that I am 5’, which you might not be able to tell over GVC.`;
+	console.log('Intro!'); 
+}
+
+function tech() { 
+	document.getElementById("inner-content").style.display = "block"; 
+	document.getElementById("para").innerHTML = `Rust, C++, C, JS`;
+	console.log('Tech!'); 
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function interests() { 
+	document.getElementById("inner-content").style.display = "block"; 
+	document.getElementById("para").innerHTML = `Violin, Piano, 
+						     AppliedCyber`; 
+	console.log('Interests!');   
+}
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function contact() { 
+	document.getElementById("inner-content").style.display = "block"; 
+	document.getElementById("para").innerHTML = 
+		`<ul style="list-style-type:none;"> 
+		<li>Email: 
+		<a href="mailto:murray22@stanford.edu"> 
+		murray22@stanford.edu 
+		</a></li>
+		<li>Linkedin:
+		<a href="https://www.linkedin.com/in/murray22/" target="_blank">
+		My LinkedIn
+		</a></li> 
+		<li>Github: 
+		<a href="https://github.com/mmurray22" target="_blank">
+		My Github
+		</a></li> 
+		</ul>`; 
```

Comment:
One thing I was thinking about is that it's a bit cumbersome to store this content in string variables in javascript when it really makes more sense to keep it in the HTML.

Let's no block on this, but consider following up to migrate this into the DOM and have these javascript functions just reveal the content that was initially hidden...

Reply:
Thank you for the suggestion, and I will make sure to incorporate that in my next PR